src/commonMain/kotlin/dev/romainguy/kotlin/math/Matrix.kt

```diff
@@ -259,6 +259,50 @@ data class Mat4(
                         degrees(-asin(z.y)), degrees(-atan2(z.x, z.z)), degrees(atan2( x.y,  y.y)))
             }
         }
+    val rotationQuaternion: Float4
```

Comment:
Properties are meant to give access to intrinsic qualities of the class, this should be a method (`toQuaternion()`) or a function (`quaternion(Mat4)`), and no need to specify `rotation` since that's what quaternions are fo.

Reply:
OK. I will move it to `fun quaternion(Mat4): Float4`

Reply:
Fixed https://github.com/romainguy/kotlin-math/pull/33/commits/fab376e40b1a1db7e48c1e8e3b517ae29a2d8a88